.github/CODEOWNERS

```
@@ -3,7 +3,7 @@
 # These owners will be the default owners for everything in the
 # repo. Unless a later match takes precedence, these owners will be
 # requested for review when someone opens a pull request.
-* @dav3r @felddy @jsf9k @mcdonnnj
+* @dav3r @felddy @jsf9k @mcdonnnj @itsmostafa
```

We alphabetize these.

```suggestion
* @dav3r @itsmostafa @felddy @jsf9k @mcdonnnj
```

---

@mcdonnnj the suggested change isnt alphabetized. I resolved it here: [13b3e78](https://github.com/cisagov/windows-server-packer/pull/10/commits/13b3e780baf6a0cfce8f0c3b40b551d0465fd21a)

---

[2acdca7](https://github.com/cisagov/windows-server-packer/pull/10/commits/2acdca785792607f177c142238ccc964ef4f5808)

---

@mcdonnnj resolved with [2facb31](https://github.com/cisagov/windows-server-packer/pull/10/commits/2facb314a95463b8478e01f6b5e40410842c104d)

---

Whoops, sorry for the mishap and thanks for catching it in your commit.